docs/Guides/Ecosystem.md

```diff
@@ -218,6 +218,12 @@ section.
   directory.
 - [`fastify-axios`](https://github.com/davidedantonio/fastify-axios) Plugin to
   send HTTP requests via [axios](https://github.com/axios/axios).
+- [fastify-aws-timestream](https://github.com/gzileni/fastify-aws-timestream)
+  Fastify plugins to managing databases, tables and querying and creating
+  scheduled queries with AWS Timestream
+- [fastify-aws-sns](https://github.com/gzileni/fastify-aws-sns) Fastify plugin
+  to AWS Simple Notification Service (AWS SNS) to coordinates and manages
+  the delivery or sending of messages to subscribing endpoints or clients
```

Comment:
```suggestion
- [fastify-aws-timestream](https://github.com/gzileni/fastify-aws-timestream)
  Fastify plugin for managing databases, tables, and querying and creating
  scheduled queries with AWS Timestream.
- [fastify-aws-sns](https://github.com/gzileni/fastify-aws-sns) Fastify plugin
  for AWS Simple Notification Service (AWS SNS) that coordinates and manages
  the delivery or sending of messages to subscribing endpoints or clients.
```

Reply:
Is that the bloody famous oxford comma, Sir Frazer Smith?
![image](https://user-images.githubusercontent.com/5059100/187688596-364fa8fe-52f5-4b01-84ef-c9c877279f23.png)

Reply:
> Is that the bloody famous oxford comma, Sir Frazer Smith? ![image](https://user-images.githubusercontent.com/5059100/187688596-364fa8fe-52f5-4b01-84ef-c9c877279f23.png)

Yeah boi 😄

Reply:
I fixed all the bugs. Thanx